wp-cli/vip-migrations.php

```diff
@@ -66,7 +66,7 @@ function dbdelta( $args, $assoc_args ) {
 		if ( in_array( $args[1], array( '', 'all', 'blog', 'global', 'ms_global' ), true ) ) {
 			$changes = dbDelta( $args[1], ! $dry_run );
 		} else {
-			$changes = dbDelta( null, ! $dry_run );
+			$changes = dbDelta( '', ! $dry_run );
```

Comment:
I think we should probably change the behaviour here. If `<tables>` is not set, we should default to `''` (this should happen very early in the command). And in this `else`, we should throw an error since an invalid `tables` value was specified.